src/Editor/Util/AssetConverter.cpp

```diff
@@ -30,6 +29,25 @@ void AssetConverter::Convert(const char * filepath, const char * destination, gl
         Log() << aImporter.GetErrorString() << "\n";
     }
 
+    if (importMaterial) {
+        /// @todo Fix Windows-specific code.
+        std::string tempMaterialFilePath = filepath;
+        std::string materialFilePath = tempMaterialFilePath.substr(0, tempMaterialFilePath.find_last_of('\\'));
+        materialFilePath += "\\";
+
+        std::string tempMaterialDestinationFilePath = destination;
+        std::string materialDestinationFilePath = tempMaterialDestinationFilePath.substr(0, tempMaterialDestinationFilePath.find_last_of('.'));
+        materialDestinationFilePath += "_";
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Variable 'materialDestinationFilePath' is assigned a value that is never used. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=cppcheck%3AunreadVariable)